README.rst

```diff
@@ -0,0 +1,7 @@
+``pyramid_favicon``
+===================
+
+``pyramid_favicon``	is an add-on for handling favicons in your Pyramid project. Normally,
+favicons are located on the root directory of a web site. However, with this add-on,
+it is possible to specify a different location for your favicon via a custom config setting.
+
```

Comment:
AFAIK, the code in this PR serves the icon from "/favicon.ico", so in other words, from root. So I don't know what you wanted to say with the paragraph above?